### PR TITLE
CLN-4776 Add integration test for only-blob-upload option for storage-deploy-action

### PR DESCRIPTION
**Pull request by mplesec**
https://leanix.atlassian.net/browse/CLN-4776

Almost forgot of this, but as promised [here](https://github.com/leanix/storage-deploy-action/pull/26) this PR adds a test for the `only-blob-upload` option.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -27,6 +27,21 @@ jobs:
...
+        id: only-blob-upload
+        uses: ./
+        with:
+          source-directory: test
+          container: storage-deploy-action-public
+          environment: test
+          only-blob-upload: true
+
+      - name: Should upload assets only to blob storage
+        if: steps.only-blob-upload.outputs != 'Deployed to Azure Blob Storage storage-deploy-action-public. Skipping deploy to Azure File Storage.'
```
Comment (dan-leanix) [inline .github/workflows/test.yml:40]:
```suggestion
        if: steps.only-blob-upload.outputs != 'Deployed to Azure Blob Storage leanixwesteuropetest. Skipping deploy to Azure File Storage.'
```

**Review by dan-leanix — CHANGES_REQUESTED** [review]

**Comment by mplesec** [comment]
@dan-leanix I see that there's a problem with the additional regions that were set up:

https://github.com/leanix/storage-deploy-action/runs/4396664548?check_suite_focus=true
```
currentRegion:  { name: 'westeurope', short: 'eu' }
Now deploying to leanixwesteuropetest!
Now deploying to Azure Blob Storage leanixwesteuropetest.
Finished deploying to region westeurope.
currentRegion:  { name: 'eastus', short: 'us' }
Storage Account leanixeastustest does not exist.
Error: ERROR: Storage account 'leanixeastustest' not found.
INFO: Command ran in 2.208 seconds (init: 0.152, invoke: 2.056)

currentRegion:  { name: 'canadacentral', short: 'ca' }
Storage Account leanixcanadacentraltest does not exist.
Error: ERROR: Storage account 'leanixcanadacentraltest' not found.
INFO: Command ran in 2.012 seconds (init: 0.150, invoke: 1.862)

currentRegion:  { name: 'australiaeast', short: 'au' }
Storage Account leanixaustraliaeasttest does not exist.
Error: ERROR: Storage account 'leanixaustraliaeasttest' not found.
INFO: Command ran in 2.461 seconds (init: 0.148, invoke: 2.313)

currentRegion:  { name: 'germanywestcentral', short: 'de' }
Storage Account leanixdetest does not exist.
Error: ERROR: Storage account 'leanixdetest' not found.
INFO: Command ran in 2.101 seconds (init: 0.159, invoke: 1.942)
```

**Comment by dan-leanix** [comment]
@mplesec That is not a problem you just need to call the storage-deploy-action with the region parameter and then this should work. ;-)

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -27,6 +27,21 @@ jobs:
...
           SAS_TOKEN: ${{ env.SAS_TOKEN }}
 
+      - name: Use storage deploy action with only-blob-upload option
+        id: only-blob-upload
+        uses: ./
+        with:
+          source-directory: test
+          container: storage-deploy-action-public
+          environment: test
+          only-blob-upload: true
```
Comment (dan-leanix) [inline .github/workflows/test.yml:37]:
```suggestion
          environment: test
          region: westeurope
          only-blob-upload: true
```

**Review by dan-leanix — CHANGES_REQUESTED** [review]

**Comment by dan-leanix** [comment]
@mplesec I would not waste any time here further. Just close the PR. If this blob only upload would not work, you would have experienced it.

Otherwise, if you want to do this. The output you reference returns an `Object`.

Furthermore, don't turn off the silent mode as otherwise we expose the SAS token to the general public here.

**Comment by mplesec** [comment]
I give up trying to get this to work. If anyone else wants to take a look at it and get the jobs to pass, please do. Closing.

What fails is this:
Screenshot 2021-12-06 at 10 54 30

**Pull request closed without merging**